bitherj/src/main/java/net/bither/bitherj/PrimerjSettings.java

```diff
@@ -54,13 +54,19 @@
 
     public static final BigInteger proofOfWorkLimit = Utils.decodeCompactBits(0x1d00ffffL);
     public static final int port = 9911;
+    public static final int testNetPort = 9913;
+    public static int getPort(boolean isTestNet){
+        if(isTestNet) return testNetPort;
+        return port;
+    }
     //    public static final int port = 8333;
     public static final long packetMagic = 0xe4e7e5e7L;
```

Comment:
iirc the magic is different for the testnet as well

Reply:
Yes, for the testnet it is 0xfbfecbc3L. Thanks for this hint.